### DISCUSSION TITLE
我发了邮件到mail@56yhz.com，退信了

### DISCUSSION THREAD

**Discussion by woodschai** [GreaterWMS/GreaterWMS#318] [General]
收件人邮件地址（mail@56yhz.com）不存在，邮件无法送达。 
host mxn.mxhichina.com[140.205.96.209] said: 554 RCPT (mail@56yhz.com) dosn't exist (in reply to RCPT TO command)

**Comment by Singosgu** [comment]
服务器正在转移国外
